_posts/2020-04-14-jump_dev_2020_cancellation.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title:  "Cancelled: JuMP-dev 2020"
+date:   2020-04-14
```

Comment:
2020-04-15

Reply:
My brain is fried after the end of this day...